kani-compiler/src/codegen_cprover_gotoc/codegen/typ.rs

```diff
@@ -127,6 +127,129 @@ impl ExprExt for Expr {
 
 /// Function signatures
 impl<'tcx> GotocCtx<'tcx> {
+    /// This method prints the details of a GotoC type, for debugging purposes.
+    #[allow(unused)]
+    pub(crate) fn debug_print_type_recursively(&self, ty: &Type) -> String {
```

Comment:
Not that it matters for this particular use case, but we should really just have a `Type` visitor

Reply:
Agreed.